scripts/cmake/macros.cmake

```diff
@@ -889,6 +889,39 @@ endfunction(add_subproject)
 # Extensions
 # ------------------------------------------------------------------------------
 
+if (ENABLE_BUILD_EFI)
```

Comment:
Is it possible to remove the ```if(ENABLE_BUILD_EFI)``` check, and then move all of the variables to either be configs in default.cmake or as local variables to an appropriate CMakeLists file under bfdriver/efi/?

I think that ideally, the only thing left in this file should be the add_efi_module macro.